src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const expected = {};
+  const propertiesAndKeys = sourceString.split(';');
+  propertiesAndKeys.pop();
+
+  propertiesAndKeys.forEach(function(element) {
+    const styles = element.split(': ');
```

Comment:
use `trim` on styles[1]
```suggestion
    const styles = element.split(':');
```

Reply:
If it would be with 2 whitespaces `:  `?